CaImAn/caiman_main.py

```diff
@@ -0,0 +1,275 @@
+# Import miscellaneous and utilities librarys
+import os
+import sys
+import h5py
+import cv2
+import numpy as np
+import logging
+from typing import Any, Dict, List, Optional, Tuple, Union, Callable
+
+sys.path.append("..")
+# Import CaimAn related utilities libraries
+import utilities as utils
+import definitions as defs
+import caiman_definitions as cm_defs
+import caiman_parameters  as cm_params
+
+# Import for CaimAn lib
+import caiman as cm
+
+# Import for PyInstaller
+from multiprocessing import freeze_support
+freeze_support()
+
+
+def main(caiman_parameters):
+    """
+    CaImAn CNMF algorithm
+    """
+
+    #os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
+
+    try:
+        cv2.setNumThreads(0)
+    except:
+        pass
+
+    # start the cluster
+    try:
+        cm.stop_server()  # stop it if it was running
+    except():
+        pass
+
+    c, dview, n_processes = cm.cluster.setup_cluster(backend="local", n_processes=None, single_thread=False)
+
+    fname_new = motion_correction(dview, caiman_parameters)
+
+    dims, T, cnm, images = cnmf(n_processes, dview, fname_new, caiman_parameters)
```

Comment:
I think we can return just `estimates` instead of `cnm`, because below we use only the `cnm.estimates`